frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick_start_hints_in_quickstart_sidepanel_ODC-4964.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Enable quickstart hints in quickstart sidepanel
+	As a user, i want to be able to click on a hint link in the quick start content and see the hint being highlighted in the UI.
+
+
+Background:
+    Given user is at developer perspective
+    And user has created cr for sample-application-quickstart quick start
+    And user has created cr for install-associate-pipeline-quickstart quick start
+    And user has created cr for explore-serverless-quickstart quick start
+    And user has created cr for add-healthchecks-quickstart quick start
+
+
+@regression @manual
+Scenario: Quick Starts hints
+   Given user is in Quick Start catalog page
+   When user clicks on sample-application quick start card
+   And user starts the tour 
+   And user clicks on the link perspective switcher in step 1
```

Comment:
And user clicks on the perspective switcher link in step 1

Reply:
Updated